api/migrations/versions/c0885e531d29_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: c0885e531d29
+Revises: 878f67285c72
+Create Date: 2019-06-06 10:24:01.869322
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c0885e531d29'
+down_revision = '878f67285c72'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('offer', sa.Column('accommodation_award', sa.Boolean(), nullable=True))
+    op.add_column('offer', sa.Column('payment_required', sa.Boolean(), nullable=False))
+    op.drop_column('offer', 'payement_required')
+    op.drop_column('offer', 'accomodation_award')
```

Comment:
does this migration run as expected? suprising :P

Reply:
oh wait, you've misspelt things?

Can you please rather use the "alter column" to change the types? Rather than misspelling correctly spelt columns?

Reply:
OH WAIT, you're correct,
previous ones were mispelt. Ignore me plz